src/index.test.ts

```diff
@@ -109,6 +109,9 @@ test('export', (t): void => {
           '@typescript-eslint/no-throw-literal': 'error',
           '@typescript-eslint/no-unnecessary-type-assertion': 'error',
           '@typescript-eslint/no-unnecessary-boolean-literal-compare': 'error',
+          '@typescript-eslint/no-unsafe-call': 'error',
+          '@typescript-eslint/no-unsafe-member-access': 'error',
```

Comment:
Hmmm, this is probably a rather big breaking change, it will basically disallow "tricking" the compiler with `as any`.

On the other hand, I think that `@ts-ignore` or `as any as Foo` are better replacements anyways so I think that I'm pro this change 👍 

We _might_ want to let others chime in first though 🤔

Reply:
Thank you, @LinusU. It's hard for me to pull the trigger on this, because it won't be my code that is breaking in several places.